test/Electrical/dc_motor.jl

```diff
@@ -0,0 +1,140 @@
+using ModelingToolkitStandardLibrary.Electrical
+using ModelingToolkitStandardLibrary.Mechanical.Rotational
+using ModelingToolkitStandardLibrary.Blocks
+import ModelingToolkitStandardLibrary
+using ModelingToolkit, OrdinaryDiffEq, Test
+# using Plots
+
+@parameters t
+
+@testset "DC motor" begin
```

Comment:
The tests for this system look very simple, aren't there more properties of the solution that could be tested?

Reply:
I will add more tests. At least the steady-state values for the armature current and angular velocity with and without load can be computed in closed form.